pkg/bpf/bpf_test.go

```diff
@@ -34,15 +34,21 @@ import (
 )
 
 func TestRestart(t *testing.T) {
-	t.Run("new start", func(t *testing.T) {
-		runTestNormal(t)
+	t.Run("new start DualEengine", func(t *testing.T) {
+		runTestNormalDualEengine(t)
```

Comment:
```suggestion
		runTestNormalDualEngine(t)
```
Spelling mistakes. Check them.